x-pack/plugins/fleet/public/applications/fleet/sections/agent_policy/create_package_policy_page/components/steps/step_define_package_policy.tsx

```diff
@@ -304,6 +315,49 @@ export const StepDefinePackagePolicy: React.FunctionComponent<{
                       />
                     </EuiFormRow>
                   </EuiFlexItem>
+
+                  {/* Output */}
+                  {canUseOutputPerIntegration && (
+                    <EuiFlexItem>
+                      <EuiFormRow
+                        label={
+                          <FormattedMessage
+                            id="xpack.fleet.createPackagePolicy.stepConfigure.packagePolicyOutputInputLabel"
+                            defaultMessage="Output"
+                          />
+                        }
+                        helpText={
+                          <FormattedMessage
+                            id="xpack.fleet.createPackagePolicy.stepConfigure.packagePolicyOutputHelpLabel"
+                            defaultMessage="Change the default output inherited from the parent agent policy. This setting changes where the integration's data is sent."
+                          />
+                        }
+                      >
+                        <EuiSelect
```

Comment:
Should we use a super select like we do for the agent policy output selector? the default value looks more explicit to me 
Currently 
<img width="440" alt="Screenshot 2024-08-12 at 10 13 11 AM" src="https://github.com/user-attachments/assets/de1fe6ae-7f38-4c8c-8177-1a83d96d93e9">
Agent policy output selector
<img width="682" alt="Screenshot 2024-08-12 at 10 13 30 AM" src="https://github.com/user-attachments/assets/4fe02648-1fc7-4629-94d2-e26c7eec5ce1">

Reply:
I can address this in a follow up PR